src/lib.rs

```diff
@@ -171,30 +176,43 @@ pub struct SystemRuntime {
 }
 
 impl SystemRuntime {
-    fn init_pid1(&self) {
+    async fn init_pid1(&self) {
         print_logo();
 
         init_pid1_logging(self.logger_level);
+        trace!("Logging started");
 
+        trace!("Configure filesystem");
         init_rootfs();
 
         // Show content of file-based kernel interface directories
         //fileio::show_dir("/dev", false);
         //fileio::show_dir("/sys", false);
         //fileio::show_dir("/proc", false);
 
-        // Implement & Discuss (TODO): init::init_network();
         // Show available network interfaces
-        // fileio::show_dir("/sys/class/net/", false);
+        //show_dir("/sys/class/net/", false);
+
+        trace!("configure network");
+        let (connection, handle, _) = new_connection().unwrap();
+        tokio::spawn(connection);
+
+        trace!("configure lo");
```

Comment:
use "looback" for logs as "lo" is subject to change in future PRs

Reply:
done